l10n_br_account_product/models/account_invoice.py

```diff
@@ -756,6 +757,21 @@ def _compute_price(self):
     freight_value = fields.Float(
         'Frete', digits=dp.get_precision('Account'), default=0.00)
     fiscal_comment = fields.Text(u'Observação Fiscal')
+    xped = fields.Char(
+        string=u"Código do Pedido (xPed)",
+        size=15,
+    )
+    nitemped = fields.Char(
+        string=u"Item do Pedido (nItemPed)",
+    )
+
+    @api.onchange("nitemped")
```

Comment:
Better to use "Integer" then and fix the tag in the pysped to "Inteiro"
https://github.com/odoo-brazil/PySPED/blob/8.0/pysped/nfe/leiaute/nfe_200.py#L1443